STMicroelectronics/STM32L4_L4+/samples/sprinkler.1/sprinkler.h

```diff
@@ -19,12 +19,14 @@ extern "C"
 #define SPRINKLER_1_PACKAGE_NAME "sprinkler"
 #define SPRINKLER_1_PACKAGE_VERSION 1
 
-  az_result sprinkler_v1i1_water_now(int32_t area, int32_t timer);
+  az_result sprinkler_v1i1_water_now(int32_t zone, int32_t timer);
```

Comment:
>v1i1 [](http://example.com/codeflow?start=22&length=4)

would this naming scheme need to change with the updated package and interface version formats? this looks like it's following the old standard

Reply:
Yes. I'll address it as soon as we define how will be the isolation between the autogenerated and developer's code. I created a Feature in DevOpts to track it.